scripts/increment-cargo-version.sh

```diff
@@ -129,7 +129,7 @@ for Cargo_toml in "${Cargo_tomls[@]}"; do
   # Set new crate version
   (
     set -x
-    sed -i "$Cargo_toml" -e "0,/^version =/{s/^version = \"[^\"]*\"$/version = \"$newVersion\"/}"
+    sed -i "$Cargo_toml" -e "s/^version = \"$currentVersion\"$/version = \"$newVersion\"/"
```

Comment:
> we've used this for years, but it started failing recently because the cargo sort v2 now moves the [package] section to the top, which cause the script to skip updating [workspace.package]

I just want to make sure I follow as I am a bit of a `sed` noob:
- The old `sed` command only matches the first instance
- The old `sed` command only matched any `version = ` whereas the new one matches `version = "CURRENT_VER"`
- cargo sort change shuffled the order, so we started matching the "wrong" line since the "wrong" line is now higher up